vfox-restServer/src/main/java/com/vfoexchange/restServer/serviceImpl/EmailServicesImpl.java

```diff
@@ -0,0 +1,62 @@
+package com.vfoexchange.restServer.serviceImpl;
+
+import com.vfoexchange.restServer.service.EmailServices;
+import com.vfoexchange.restServer.util.AppUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Service;
+
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import java.io.UnsupportedEncodingException;
+import java.util.Properties;
+
+@Service("emailServices")
+public class EmailServicesImpl implements EmailServices {
+
+    @Autowired
+    private Environment environment;
+
```

Comment:
Remove unnecessary blank line.

Reply:
Unnecessary blank line removed and committed.